pkg/imageregistry/imageregistry.go

```diff
@@ -134,8 +134,23 @@ func (builder *Builder) GetManagementState() (v1.ManagementState, error) {
 	return builder.Object.Spec.ManagementState, nil
 }
 
+// GetStorageConfig fetches imageRegistry Storage configuration.
+func (builder *Builder) GetStorageConfig() (imageregistryv1.ImageRegistryConfigStorage, error) {
+	if valid, err := builder.validate(); !valid {
+		return imageregistryv1.ImageRegistryConfigStorage{}, err
+	}
+
+	glog.V(100).Infof("Getting imageRegistry ManagementState configuration")
```

Comment:
q. Isn't it Storage configuration, rather than ManagementState?

Reply:
sure, sorry, typo